model_compression_toolkit/target_platform_capabilities/tpc_models/get_target_platform_capabilities.py

```diff
@@ -25,7 +23,7 @@
     tpc_dict as qnnpack_tpc_dict
 from model_compression_toolkit.target_platform_capabilities.constants import DEFAULT_TP_MODEL, IMX500_TP_MODEL, TFLITE_TP_MODEL, QNNPACK_TP_MODEL,  LATEST
 
-tpc_dict = {DEFAULT_TP_MODEL: default_tpc_dict,
+tpc_dict = {DEFAULT_TP_MODEL: imx500_tpc_dict,
```

Comment:
We may consider what happens if someone asks for V1 in imx500. He will get different TPC (he got the 'default' one until now).